### ISSUE TITLE
/logout returns blank page

### ISSUE THREAD

**Issue by datitisev** [flarum/core#1575]
## Bug Report

**Current Behavior**
`/logout` returns a blank page and gives no indication that the user has been logged out / is already logged out.

**Steps to Reproduce**
- Go to `/logout`
- Log out when signed in

**Expected Behavior**
Redirects to previous page or home page after being logged out.

**Screenshots**
![image](https://user-images.githubusercontent.com/6401250/45918078-7b973100-be4f-11e8-8a3e-727939240c3b.png)

**Environment**
- Flarum version: dev-master (https://github.com/flarum/core/commit/0fb9aa3940a8fbb11905e60cc37ca9cdcef62589)
- Webserver: nginx
- PHP version: 7.1.21
- Browser: Chrome 70

Cannot give output of `php flarum info`, https://github.com/flarum/flarum/commit/655aa9cea91dab571b67d917f37cc95ce3615285 removed all commands except `help`, `install`, and `list`.

**Comment by franzliedke** [comment]
I can reproduce. It's a HTTP 302 response with an empty `Location` header. :thinking: 

> Cannot give output of `php flarum info`

That should only be the case when there is no config.php file. And that behavior should not be changed by the commit you referenced.

**Comment by datitisev** [comment]
@franzliedke Hm, you are correct. Setting `url` to the correct value did the proper redirect.
I installed fresh beta 8 instance using the web installer.

---
About the `flarum` binary, I looked at https://github.com/flarum/core/commit/4ed1c7a1bb31092258e2ea5ebb27fc4f5067c1e7 and saw that it shouldn't be affected, but I do not have the info command.
```sh
$ php flarum list
Flarum 0.1.0-beta.7

Usage:
  command [options] [arguments]

Options:
[ • • •]

Available commands:
  help     Displays help for a command
  install  Run Flarum's installation migration and seeds
  list     Lists commands
$ ls config.php
config.php
```

**Comment by franzliedke** [comment]
This will be fixed by #1453.
